ios/FluentUI/Drawer/DrawerPresentationController.swift

```diff
@@ -315,8 +318,12 @@ class DrawerPresentationController: UIPresentationController {
         var contentFrame = bounds.inset(by: marginsForContentView())
 
         var contentSize = presentedViewController.preferredContentSize
+        let landscapeMode = contentFrame.width > contentFrame.height
+
         if presentationDirection.isVertical {
-            if contentSize.width == 0 || traitCollection.horizontalSizeClass == .compact {
+            if contentSize.width == 0 ||
+                (traitCollection.userInterfaceIdiom == .phone && landscapeMode && shouldUseWindowFullWidthInLandscape) ||
```

Comment:
what about iPhones in landscape mode that are considered compact, shouldn't they be presented as sliders regardless of shouldUseWindowFullWidthInLandscape?

Reply:
It is shown as a slider in landscape. the request from List team was only on landscape it should show the "preferred" width. here is a screenshot of iPhoneSE
![Simulator Screen Shot - iPhone SE (1st generation) - 2020-06-02 at 19 45 26](https://user-images.githubusercontent.com/20715435/83591614-30329280-a50d-11ea-83a4-edec20ee6a98.png)